### PR TITLE
Change to junit5 methods for some tests

### PR DESCRIPTION
**Pull request by Laurens-makel**

### CODE REVIEW

**Review by jkosternl — APPROVED** [review]
Hopefully this PR does not double the work Yannick is already doing in my open PRs already. @Laurens-makel, please check his work, if you create more PRs regarding this topic.

File: core/src/test/java/org/frankframework/statistics/StatisticsKeeperTest.java [hunk 1]
```diff
@@ -159,12 +158,12 @@ public void testInterval() {
...
-		assertEquals(sk.getIntervalItemName(0), 0L,   sk.getIntervalItemValue(0)); // count
-		assertEquals(sk.getIntervalItemName(1), null, sk.getIntervalItemValue(1)); // min
-		assertEquals(sk.getIntervalItemName(2), null, sk.getIntervalItemValue(2)); // max
-		assertEquals(sk.getIntervalItemName(3), null, sk.getIntervalItemValue(3)); // avg
-		assertEquals(sk.getIntervalItemName(4), 0L,   sk.getIntervalItemValue(4)); // sum
-		assertEquals(sk.getIntervalItemName(5), 0L,   sk.getIntervalItemValue(5)); // sumSq
+		assertEquals(0L,   sk.getIntervalItemValue(0), sk.getIntervalItemName(0)); // count
+		assertEquals(null, sk.getIntervalItemValue(1), sk.getIntervalItemName(1)); // min
+		assertEquals(null, sk.getIntervalItemValue(2), sk.getIntervalItemName(2)); // max
+		assertEquals(null, sk.getIntervalItemValue(3), sk.getIntervalItemName(3)); // avg
```
Comment (tnleeuw) [inline core/src/test/java/org/frankframework/statistics/StatisticsKeeperTest.java:164]: What about using `assertNull()`?

File: core/src/test/java/org/frankframework/util/LockerTest.java [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.fail;
 import static org.junit.Assume.assumeFalse;
```
Comment (tnleeuw) [inline core/src/test/java/org/frankframework/util/LockerTest.java:9]: Still using the JUnit4 `assumeFalse()` method and Hamcrest Matchers that are JUnit4 compatible.
However I'm aware by now that this is not a simple change.

**Review by tnleeuw — CHANGES_REQUESTED** [review]
I don't know how much this doubles with work done by @Yale96  but I see the tests still use JUnit4 `@Test` annotations instead of JUnit5 `@Test` annotations.

Please change those annations as well, otherwise I do not see a lot of point in just changing only the assertions.

**Comment by Laurens-makel** [comment]
Thanks for the review.

I'm planning to work on the feedback in the next couple of days. If a problem arises with the Hamcrest Matchers, I think I'll keep it out of scope for this PR since this was meant as a quick & easy win.

**Comment by tnleeuw** [comment]
> Thanks for the review.
> 
> I'm planning to work on the feedback in the next couple of days. If a problem arises with the Hamcrest Matchers, I think I'll keep it out of scope for this PR since this was meant as a quick & easy win.

That's fine, I just wanted to highlight it!

**Pull request closed without merging**